charts/opentelemetry-operator/values.yaml

```diff
@@ -124,6 +124,9 @@ manager:
       # Specifies whether recommended updates are applied when a Pod is started and whether recommended updates
       # are applied during the life of a Pod. Possible values are "Off", "Initial", "Recreate", and "Auto".
       updateMode: Auto
+      # Minimal number of replicas which need to be alive for Updater to attempt pod eviction.
+      # Only positive values are allowed. The default is 2.
+      minReplicas: ""
```

Comment:
linting fails as this is defined as string vs integer in the schema.

Reply:
changed it. thanks